### PR TITLE
[UK] Translate global definitions

### PR DESCRIPTION
**Pull request by brain-geek**
I've noticed there are some Ukrainian parts not currently updated. :)

Updating.

Hey, @doroshenkoss , could you review this? It would be awesome.

Thanks!

### CODE REVIEW

File: _data/locales/uk.yml [hunk 1]
```diff
@@ -5,27 +5,27 @@ project:
...
+
 translation_report:
-  section: Section
-  original_title: Original Title
-  translation_title: Translation Title
-  original_version: Original Version
-  translation_version: Translation Version
-  version_severity: Version Severity
+  section: Розділ
+  original_title: Оригінальна Назва
```
Comment (doroshenkoss) [inline _data/locales/uk.yml]: I think capital letter not needed here - "Назва"

File: _data/locales/uk.yml [hunk 2]
```diff
@@ -5,27 +5,27 @@ project:
...
+  translation_title: Перекладена Назва
```
Comment (doroshenkoss) [inline _data/locales/uk.yml]: Same

File: _data/locales/uk.yml [hunk 3]
```diff
@@ -5,27 +5,27 @@ project:
...
-  patch: "Several patches were applied to the original lesson since the last update."
-  none: "This translation is up to date."
-  error: "There is an error in the version definition of this chapter."
-  
+  outdated: "Деякі частини цього перекладу можуть бути застарілими."
+  major: "Декілька великих змін було додано до оригіналу цього уроку з останнього оновлення."
+  minor: "Декілька невеликих змін було додано до оригіналу цього уроку з останнього оновлення."
+  patch: "Декілька косметичних змін було додано до оригіналу цього уроку з останнього оновлення."
+  none: "Переклад актуальний."
+  error: "В дефініції уроку знайдено помилку."
```
[the remark this answers is not in the document]
Comment (brain-geek) [reply _data/locales/uk.yml]: I like the first version more, applied.

**Comment by brain-geek** [comment]
@doroshenkoss Thanks for the review! I've applied your suggestions. Could you look at it once again?

**Pull request merged**